recipes/openmmtorchplugin/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set name = "openmmtorchplugin" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SCMusson/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 1a9197c32209eaa37b9dcb1a7ef6ed30841ea5beed124b902485c20f5dd92021
+
+build:
+  number: 0
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}   # [cuda_compiler_version != "None"]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [cuda_compiler_version == "None"]
+  skip: true  # [not linux or (linux and cuda_compiler_version in (undefined, 'None', '10.2'))]
```

```suggestion
  skip: true  # [cuda_compiler_version in (undefined, 'None', '10.2')]
  skip: true  # [not linux]
```
I believe that multiple skips is equivalent to logical OR.